service/storage_proxy.cc

```diff
@@ -493,47 +539,63 @@ class storage_proxy::remote {
         co_return netw::messaging_service::no_wait();
     }
 
+    future<rpc::no_wait_type> receive_hint_mutation_handler(
+        const rpc::client_info& cinfo, rpc::opt_time_point t,
+        frozen_mutation in, inet_address_vector_replica_set forward, gms::inet_address reply_to,
+        unsigned shard, storage_proxy::response_id_type response_id,
+        rpc::optional<std::optional<tracing::trace_info>> trace_info,
+        rpc::optional<std::optional<fencing_token>> fence) {
+        return receive_mutation_handler(_sp._hints_write_smp_service_group, cinfo, t, std::move(in),
```

Comment:
Received hint mutation is handled in the same way as regular mutation. This means for forwarding we will call `send_mutation`, not `send_hint_mutation`. Not sure if this is what we need here.

Reply:
There is no forwarding for hints mutations since they are send with empty forwarding list.

Reply:
ok